operator/params.yaml

```
@@ -143,6 +143,10 @@ parameters:
     description: "The JMX port that will be used to interface with the Cassandra application."
     default: "7199"
 
+  - name: JMX_LOCAL_ONLY
+    description: "If true, the JMX port will only be opened on localhost and not be available to the cluster"
+    default: "false"
```

Let's use the more restrictive local-only configuration as default.
```suggestion
    default: "true"
```

---

Yeah, I thought about that this morning as well. This will be a prerequisite for enabling backup & restore though, it might be a good idea to have some feature in KUDO that allows checking of depending parameters, i.e. if you have `ENABLE_BACKUP_RESTORE=true` you also need to have `JMX_LOCAL_ONLY=false`.

Changed it to default false anyway.